test/src/audits/performance/user-timing.js

```diff
@@ -18,23 +18,16 @@
 const Audit = require('../../../../src/audits/performance/user-timings.js');
 const assert = require('assert');
 const traceContents = require('./trace-user-timings.json');
-
 /* eslint-env mocha */
+
 describe('Performance: user-timings audit', () => {
   it('fails gracefully', () => {
     const output = Audit.audit({});
     assert.equal(output.value, -1);
   });
 
-  it('processes a trace file for user timing data', () => {
+  it('evaluates valid input correctly', () => {
     const output = Audit.audit({traceContents});
-    assert.equal(output.value, 1);
-    assert.equal(output.extendedInfo.value[0].name, 'elapsed');
-  });
-
-  it('handles valid traces with no user timing data', () => {
-    const output = Audit.audit({traceContents: traceContents.slice(0, 1)});
-    assert.equal(output.value, 0);
-    assert.equal(output.extendedInfo.value.length, 0);
+    assert.equal(output.value, 2);
```

Comment:
when I run this test, `navigationStart` isn't being found in the trace, and the audit output looks like:
```
{
  "value": 2,
  "extendedInfo": {
    "formatter": "userTimings",
    "value": [
      {
        "name": "mark_test",
        "isMark": true,
        "args": {},
        "startTime": null
      },
      {
        "name": "measure_test",
        "isMark": false,
        "args": {},
        "startTime": null,
        "duration": 1000.964999973774,
        "endTime": null
      }
    ]
  },
  "name": "user-timings",
  "category": "Performance",
  "description": "User Timing marks and measures"
}
```
it's those `startTime: null`/`endTime: null` that I think the test should be making sure isn't happening here